scripts/pages/pgSignUp.ts

```diff
@@ -145,8 +140,8 @@ export default class PgSignUp extends withDismissAndBackButton(PgSignUpDesign) {
             Application.statusBar.visible = true;
             Application.statusBar.backgroundColor = Color.WHITE;
         }
-        this.initBackButton(this.router, {
-            color: themeService.getNativeStyle('.sf-headerBar.main').itemColor
+        this.initDismissButton(this.router, {
+            color: themeService.getNativeStyle('.sf-headerBar.transparent.white').itemColor
```

Comment:
Maybe we should declare itemColor value top on the page class. @suleymanbasbug

```ts
const itemColor = themeService.getNativeStyle('.sf-headerBar.transparent.white').itemColor;

export default PgSignUp {
// ...
this.initDismissButton(this.router, { color: itemColor
// ...
}
```

Reply:
ok.